README.md

```diff
@@ -62,22 +63,23 @@ ssh localhost -p 1415
 
 ## ⌚️ Performance
 
-This package is not optimized for stability or speed.  
+This package is not optimized for stability ~~or speed~~.
 
 _Setup_
+
 ```bash
 # Terminal 0 - Netcat listening
-nc -l 1234 > /dev/zero
+nc -l 1234 > /dev/null
 
 # Terminal 1 - Exit Node
-tcp-over-http exit --target-host locahost --target-port 1234
+tcp-over-http exit --target-addr localhost:1234
 
 # Terminal 2 - Entry Node
 tcp-over-http entry --target-url http://localhost:8080/
 
-# Terminal 3 - Sending random data
+# Terminal 3 - Sending \0 data
 # Using pipeviewer (pv) to see current data rate
-time dd if=/dev/random bs=1M count=1024 | pv | nc localhost 1415 -q 0
+time cat /dev/zero | pv | nc localhost 1415
 ```
 
-### 🏅 Result: 6MiB/s
+### 🏅 Result: $$$$MiB/s
```

Comment:
`600MiB/s` on my machine.  We should add it here as a rough estimate.

Reply:
overhead in % would be best i think

Reply:
Overhead in performance is pretty much I assume. If you can do a few gigs so, than 600MiB/s does not sound like much and a number like `15% of original speed` is not much help.  

Overhead in data however would be pretty interesting and a real number to show.